app/src/main/AndroidManifest.xml

```diff
@@ -42,38 +37,52 @@
             android:value="@string/map_key" />
 
         <activity
-            android:name=".chat.ChatActivity"
+            android:name=".view.chat.ChatActivity"
             android:label="@string/title_activity_chat" />
-
         <activity
-            android:name=".HomeScreenActivity"
+            android:name=".view.home.HomeScreenActivity"
             android:configChanges="orientation|keyboardHidden|screenSize"
             android:theme="@style/Theme.movemeet.noActionBar">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
+
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
-
         <activity
-            android:name=".RegisterActivity"
-            android:parentActivityName=".HomeScreenActivity"
+            android:name=".view.home.RegisterActivity"
+            android:parentActivityName=".view.home.HomeScreenActivity"
+            android:theme="@style/Theme.movemeet">
+            <meta-data
+                android:name="android.support.PARENT_ACTIVITY"
```

Comment:
Why is the purpose of having this metadata to specify the parent ? Is it for the back button ?

Reply:
This is the same question I asked to Jean-Luc at the beginning of the week. This is apparently for having the default bar at the top of the activity when we are in an activity where we can't access the menu. This metadata is in fact related to our custom "back button".